vissat_config.json

```diff
@@ -0,0 +1,26 @@
+{
+  "dataset_dir": "/path/to/image/data",
```

Comment:
rename to `example_vissat_config.json` and make this a valid json file.  Add a readme RST file that documents what goes into this file.